notebooks/119-tflite-to-openvino/119-tflite-to-openvino.ipynb

```diff
@@ -9,9 +9,9 @@
     "\n",
```

Comment:
<p>Optionally, we can apply compression to the FP16 model weights, using the <code>compress_to_fp16=True</code> option and integrate preprocessing using this approach.  - looks like could be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>fixed, thank you!</p>